### DISCUSSION TITLE
Adding an SSL Cert

### DISCUSSION THREAD

**Discussion by jerquiaga** [makeplane/plane#1215] [Q&A]
Any directions for adding an SSL Cert to the Plane self-hosting setup?

**Comment by georg-schwarz** [comment]
I've gone down the way of setting up self-hosted Plane with SSL via docker-compose.
Here a summary of how I succeeded: https://georg-schwarz.com/2023/07/14/nginx-reverse-proxy-with-ssl-forwarding-to-your-application/
